src/frontend/src/pages/PPQFormPage/PPQFormPage.tsx

```diff
@@ -134,32 +135,16 @@ const PPQFormPage = () => {
               />
             </div>
             <div className="row">
```

Comment:
You may not need this div as well. Row without col is useless and uses extra margins.

Reply:
Ignore this. You're using it as an optional class. My bad.

However, if you look at the classes in the developer console, there are two `form-group`. One from your line #138 and other from Dropdown common component.

You can remove line #138 instead.

Reply:
done